Gui/DopeSheetView.cpp

```diff
@@ -3171,6 +3171,8 @@ DopeSheetView::initializeGL()
         return;
     }
 
+    makeCurrent();
```

Comment:
same as above, no need to call makeCurrent, according to the doc

Reply:
You're right, I think it's something I left while debugging